File: exercism/python/darts/darts.py
import math

def score(x, y):
    result = 0
    rad = math.sqrt(x**2 + y**2)
    if rad > 5 and rad <= 10:
       result = 1
    elif rad > 1 and rad <= 5:
        result = 5
    elif rad >= 0 and rad <=1:
        result = 10
    
    return result

File: exercism/typescript/darts/darts.ts
export const score = (x: number, y: number): number => {
    let result = 0;
    const rad = (x === 0) ? Math.abs(y) : (y === 0) ? Math.abs(x) : Math.sqrt(x ** 2 + y ** 2);
    
    if (rad > 5 && rad <= 10) {
        result = 1;
    } else if (rad > 1 && rad <= 5) {
        result = 5;
    } else if (rad >= 0 && rad <= 1) {
        result = 10;
    }
    
    return result;
};